app/src/main/java/com/android/periodpals/ui/timer/TimerScreen.kt

```diff
@@ -113,99 +139,107 @@ fun TimerScreen(
 
       // Displayed text
       Text(
-          text = correct_displayedText(isTimerRunning),
+          text =
+              if (!isRunning) DISPLAYED_TEXT_START
+              else if (remainingTime < COUNTDOWN_6_HOURS / 2) DISPLAYED_TEXT_EARLY
+              else DISPLAYED_TEXT_LATE,
```

Comment:
Would it be possible to replace this with a `when` instead of these `if-else if-else`?
```suggestion
              when {
                !isRunning -> DISPLAYED_TEXT_START
                remainingTime < COUNTDOWN_6_HOURS / 2 -> DISPLAYED_TEXT_EARLY
                else -> DISPLAYED_TEXT_LATE
              },
```

Reply:
Sure, it would be cleaner :)